import.html

```diff
@@ -98,15 +98,14 @@
                 state: 0,
                 contributors: 0,
                 progress: 0,
-
             }
-
-
            importRef.push().set({
                 project: project,
                 kml: kml,
                 key: submissionKey,
+				zoomlevel: zoomlevel,
```

Comment:
It would probably improve readability to keep this at the same intent level as other variables here.